### DISCUSSION TITLE
If the checkbox will be unchecked by add 'checked=false' in clarity input label?

### DISCUSSION THREAD

**Discussion by yandong01** [vmware/clarity#5912] [Q&A] [answered]
If the checkbox will be unchecked by add 'checked=false' in clarity input label, like below?
`  <input type="checkbox" clrCheckbox name="options"  value="option1" [(ngModel)]="options.option1" checked=false />`

**Comment by gnomeontherun** [comment, marked as answer]
Angular checkboxes look at the value of the `ngModel` so the `checked` attribute should not be used. You modify the value of a checkbox by changing the value attached with ngModel.
